database/database.go

```diff
@@ -0,0 +1,53 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	_ "modernc.org/sqlite"
+)
+
+const defDBFile = "./scheduler.db"
+
+func InitDatabase() (*sql.DB, error) {
+	dbFile := os.Getenv("TODO_DBFILE")
+	if dbFile == "" {
+		dbFile = defDBFile
+	}
+
+	db, err := sql.Open("sqlite", dbFile)
+	if err != nil {
+		return nil, fmt.Errorf("can't open database: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return nil, fmt.Errorf("can't open database: %w", err)
```